std_msgs/Header header
TrackedObject3D[] objects    # Array of 3D tracked objects
int32 total_tracks          # Total number of active tracks
int32 high_quality_tracks   # Number of high-quality tracks
int32 moving_objects        # Number of moving objects
float32 processing_time_ms  # Processing time in milliseconds